resume.tex

```diff
@@ -58,7 +55,7 @@
     \skillsCategory{Languages}{
       Java,
       JavaScript,
-	  Typescript,
+      Typescript,
```

Comment:
Whole batch of "tabs --> spaces" changes.

Reply:
Thanks!